### ISSUE TITLE
JetForms add date-range picker

### ISSUE THREAD

**Issue by ghost** [CrocoBlock/suggestions#6336]
**Please, describe the feature request, making it clear to understand.**

Add the ability to have a daterange picker as a type of field in a jet form

**Add an example of where this feature is already implemented. **

Allow users select a date range uppon making an inquiry for a booking, rental request or general availability request.

**Comment by BaronVonDerpsalot** [comment]
Yup, lots of use-cases here, and lots of people who would expect this to be part of any premium form offering, I think.
